README.md

```diff
@@ -1,38 +1,38 @@
+That is the reason we came up with `Flashback`, a MongoDB benchmark framework that allows us to benchmark with "real" queries. It is comprised of a set of scripts that fall into the 2 categories:
 
-1. records the operations(_ops_) that happens during a stretch of time;
+1. records the operations(_ops_) that occur during a stretch of time;
 2. replays the recorded ops.
 
-The two parts do not necessarily couple with each other and can be used independently for different purposes.
+The two parts are not tied to each other and can be used independently for different purposes.
 
 # How it works
 
 ## Record
 
-How can you know which ops are performed by MongoDB? There are a lot of ways to do this. But in Flashback, we record the ops by enabling MongoDB's [profiling](http://docs.mongodb.org/manual/reference/command/profile/).
+How do you know which ops are performed by MongoDB? There are a lot of ways to do this. But in Flashback, we record the ops by enabling MongoDB's [profiling](http://docs.mongodb.org/manual/reference/command/profile/).
 
-By setting the profile level to _2_ (profile all ops), we'll be able to fetch the ops information that is detailed enough for future replay -- except for _insert_ ops.
+By setting the profile level to _2_ (profile all ops), we'll be able to fetch the ops information detailed enough for future replay -- except for _insert_ ops.
 
-MongoDB intentionally avoids putting insertion details in profiling results because they don't want to have the insertion being written several times. Luckily, if a MongoDB instance is working in a "replica set", then we can complete the missing information through _oplog_.
+MongoDB intentionally avoids putting insertion details in profiling results because they don't want to have the insertion being written several times. Luckily, if a MongoDB instance is working in a "replica set", then we can complete the missing information by reading the _oplog_.
 
 Thus, we record the ops with the following steps:
 
-1. Script starts two threads to pull the profiling results and oplog entries for collections that we are interested in. 2 threads are working independently.
+1. The script starts two threads to pull the profiling results and oplog entries for collections that we are interested in. The 2 threads work independently.
 2. After fetching the entries, we'll merge the results from these two sources and have a full pictures of all the operations.
 
-NOTE: If the oplog size is huge, fetching the first entry from oplog may take a long time (several hours) because oplog is unindexed. After that it will catch up with present time quickly.
+NOTE: If the oplog size is huge, fetching the first entry from oplog may take a long time (several hours) because the oplog is unindexed. After that it will catch up to present time quickly.
```

Comment:
Actually this statement is no longer accurate since we set the oplogReplay option in the cursor. http://docs.mongodb.org/manual/reference/method/cursor.addOption/#DBQuery.Option.oplogReplay 
https://github.com/ParsePlatform/flashback/blob/master/record/utils.py#L60

I'll remove this line in a later diff.